This file is part of MXE.
See index.html for further information.

Disable examples, test, and documentation in the CMakeLists.txt file.
(The file unfortunately does not provide switches for this purpose.)
diff -uNr flann-1.7.1-src/CMakeLists.txt flann-1.7.1-src.patched/CMakeLists.txt
--- flann-1.7.1-src/CMakeLists.txt	2011-12-23 12:16:09.000000000 +0100
+++ flann-1.7.1-src.patched/CMakeLists.txt	2012-10-01 12:32:59.865199865 +0200
@@ -143,9 +143,9 @@
 
 add_subdirectory( cmake )
 add_subdirectory( src )
-add_subdirectory( examples )
-add_subdirectory( test )
-add_subdirectory( doc )
+#add_subdirectory( examples )
+#add_subdirectory( test )
+#add_subdirectory( doc )
 
 
 # CPACK options

